site/content/docs/5.3/utilities/link.md

```diff
@@ -87,14 +87,20 @@ Just like the `.link-opacity-*-hover` utilities, `.link-offset` and `.link-under
 
 [Colored link helpers]({{< docsref "/helpers/colored-links/" >}}) have been updated to pair with our link utilities. Use the new utilities to modify the link opacity, underline opacity, and underline offset.
 
-<!--Boosted mod: feature limited to primary and light, we don't loop over "theme-colors"-->
+<!--Boosted mod: use `contrast_color` as background ensure a good contrast-->
 {{< example >}}
-<p><a href="#" class="link-primary link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Primary link</a></p>
-<p><a href="#" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Light link</a></p>
+{{< colored-links.inline >}}
+  {{- range (index $.Site.Data "theme-colors") }}
+    <p><a href="#" class="link-{{ .name }}{{ with .contrast_color }} bg-{{ . }}{{ end }} link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">{{ .name | title }} link</a></p>
```

Comment:
Indentation issue in the documentation:
![Screenshot 2023-07-11 at 09 36 11](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/17381666/6fb71db4-d6df-48f6-a86b-843276a653d8)